script/Deployment.s.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+
+// Import contracts
+import {DeterministicFactory_v1} from "src/DeterministicFactory_v1.sol";
+
+contract DeploymentScript is Script {
+    uint deployerPrivateKey = vm.envUint("DEPLOYER_PRIVATE_KEY");
+    address deployer = vm.addr(deployerPrivateKey);
+
+    address multisig = vm.envAddress("MULTISIG_ADDRESS");
+
+    //--------------------------------------------------------------------------
+    /// @notice Deploys the Deterministic Factory contract
+    /// @return deterministicFactory The address of the deployed Deterministic Factory
+    function run() public virtual returns (address deterministicFactory) {
+        console2.log(
+            "-----------------------------------------------------------------------------"
+        );
+        console2.log("Deploy Inverter Deterministic Factory...");
+
+        if (multisig == address(0)) {
+            console2.log("No MULTISIG_ADDRESS set, using deployer as owner.");
+            multisig = deployer;
```

Comment:
Yes, I did that because the Contract will not be deployed via Foundry on live networks, only for testing. The way it will be deployed is via `PrintBytecode` and then as a tx in safe Safe UI. That's why I don't revert in the `Deployment` script, as that's just for testing.